packages/core/src/services/db/backends/ephem-db.provider.ts

```diff
@@ -88,9 +93,9 @@ export class EphemDBProvider implements BaseDBProvider {
    * Should be more efficient than simply calling `set` repeatedly.
    * @param objects A series of objects to put into the database.
    */
-  public async bulkPut(objects: DBObject[]): Promise<void> {
+  public async batch(objects: DBObject[]): Promise<void> {
     for (const object of objects) {
-      await this.set(object.key, object.value)
+      await this.put(object.key, object.value)
```

Comment:
Should we check the operation here (`put` or `del`)?

Reply:
For sure we should. However, I just didn't want to break the dependencies for ephemDB & so I kept the implementation of batch as being what `bulkPut` used to be. Basically I'm trying to make minimal changes which would allow me to integrate the operator using the DB service